src/main/resources/META-INF/plugin.xml

```diff
@@ -227,6 +212,27 @@
             <!-- In $default keymap `control` key (Windows/Linux) corresponds to `cmd` key on macOS -->
             <keyboard-shortcut keymap="$default" first-keystroke="control alt shift M" />
         </action>
+
+        <action id="GitMachete.NextBranchCheckoutAction"
+                class="com.virtuslab.gitmachete.frontend.actions.traverse.NextBranchCheckoutAction">
+            <add-to-group group-id="GitRepositoryActions" anchor="last"/>
+            <keyboard-shortcut keymap="$default" first-keystroke="control alt shift T" second-keystroke="control shift L" />
+            <!--  currently, the second key stroke is not getting triggered. suggestions welcome-->
+        </action>
+
+        <action id="GitMachete.PreviousBranchCheckoutAction"
+                class="com.virtuslab.gitmachete.frontend.actions.traverse.PreviousBranchCheckoutAction">
+            <add-to-group group-id="GitRepositoryActions" anchor="last"/>
+            <keyboard-shortcut keymap="$default" first-keystroke="control alt shift T" second-keystroke="shift B" />
```

Comment:
Huh `Ctrl Alt Shift T` is somewhat unfortunate as it opens PRs on Linux 😅 what's the Mac equivalent? (you need to be logged in to GitHub in IntelliJ probably to see that tool window tho)

Reply:
How about `Ctrl Alt Shift G` however? Doesn't seem to correspond to anything important (`Tx Isolation` in `Database` menu xD). 

Note the comment in line 212 ofc (`<!-- In $default keymap `control` key (Windows/Linux) corresponds to `cmd` key on macOS -->`)

Reply:
Perhaps, for a more convenient use, we can put, `Ctrl+alt+Shift+G+N` for next `Ctrl+alt+Shift+G+P` for Previous
`Ctrl+alt+Shift+G+U` for up?

Reply:
Unfortunately, not allowed :(

Reply:
> Perhaps, for a more convenient use, we can put, Ctrl+alt+Shift+G+N for next Ctrl+alt+Shift+G+P for Previous
Ctrl+alt+Shift+G+U for up?

That would be awesome... okay so what was specifically not working? Is `Ctrl+Alt+Shift+G` somehow forbidden as first keystroke of double-stroke shortcut? In general, if you could research & post what combinations are allowed in double-stroke shortcuts (maybe it's documented somewhere by JetBrains?)

Reply:
Ok, seen the response below, nvm ;D